micromasters/urls.py

```diff
@@ -52,6 +53,8 @@
         name='financial_aid_action'),
     url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
         GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/$',
+        DocumentsSentView.as_view(), name='financial_aid_documents_sent'),
```

Comment:
Yep, that's correct.  The `UpdateAPIView` implements these requests as a `PUT` request, so it should behave the way you're describing already, unless I'm misunderstanding?